### PR TITLE
Added more headings font-weight options, #379

### PR DESCRIPTION
**Pull request by kkoryaka**
- Added more Public Sans font files to the fonts folder (font weight variations)
- Added references to those additional fonts files in _fonts.scss
- Added [sample page](https://1-0-6-headings.pr.designsystem.webstandards.ca.gov/pr/1-0-6-headings/typography/) with real time content that is displaying headings and their font-weight variations

NOTE: THis is font exploration is case study that will be presented to DCC customer. New headings font weight applied to the headings in the [sample page](https://1-0-6-headings.pr.designsystem.webstandards.ca.gov/pr/1-0-6-headings/typography/) only and is not applied globally yet. 

Figma headings font exploration reference screenshot:
Screen Shot 2022-04-13 at 11 47 31 AM

### CODE REVIEW

**Comment by aaronhans** [comment]
Nice @kkoryaka! Will be cool to split this out into a separate typography component soon

**Review by kmbrlygl — CHANGES_REQUESTED** [review]
One suggested style addition to h5. Thanks!

File: docs/pages/typography.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+description: Headings font size exploration.
+---
+
+
+<style>
+h1 {font-weight:  var(--font-weight-2, 200);}
+h2 {font-weight:  var(--font-weight-3, 300);}
+h3 {font-weight:  var(--font-weight-6, 600);}
+h4 {font-weight:  var(--font-weight-5, 500);}
+h5 {font-weight:  var(--font-weight-8, 800);}
```
[the remark this answers is not in the document]
Comment (kkoryaka) [reply docs/pages/typography.md:12]: @kmbrlygl  Just added it in my latest commit

**Review by kmbrlygl — APPROVED** [review]
Looks great! Thanks Konstantin

**Comment by yesenia-gtz** [comment]
Good to go @timothyshambra, Thank you!

**Pull request merged**